SubmittedAssignments/Iram_Sayyad/preview.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="./css/style.css">
+    <link rel="stylesheet" type="text/css" href="./css/bootstrap.min.css">
+</head>  
+<body>
+	<!--Certificate Wrapper-->
+	<div class="certificate-wrapper">
+        <!--watermark-->
+        <div class="watermark"></div>
+		<!--Certificate background-->
+		<div class="certificate-background">
+            <div class="certificate-border-outer"></div>
+            <div class="certificate-border-inner"></div>
+			<div class="triangle-top-right-corner"></div>
+            <div class="triangle-top-right-shadow"></div>
+            <div class="triangle-top-left-corner"></div>
+            <div class="triangle-top-left-shadow"></div>
+            <div class="triangle-bottom-right-corner"></div>
+            <div class="triangle-bottom-right-shadow"></div>
+            <div class="triangle-bottom-left-corner"></div>
+            <div class="triangle-bottom-left-shadow"></div>
+            <div class="line-bottom-left"></div>
+            <div class="line-bottom-right"></div>
+		</div>
+
+		<!--Content of Certificate-->
+		<div class="certificate-content">
+            <div class="title">CERTIFICATE</div>
+            <div class="sr-no">Certificate Number: 1234</div>
+			<div class="date">Date: 26-01-2024</div>
+            <div class="subtitle">
+                <img src="./img/RibbonImage.jpg" alt="ribbon" width="400mm" height="100mm">
+                <p>OF ACHIEVEMENT</p>
+            </div>
+            <div class="certificate-logo">
+				<img src="./img/certificate-logo.png">
+			</div>
+
+            <!-- Content -->
+            <div class="initial-content">This Certificate is proudly presented to</div>
+            <div class="candidate-name">Name Surname</div>
+            <div class="main-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi eget euismod ipsum.<br>Ut in mauris ut justo laoreet malesuada. In hac habitasse platea dictumst.Curabitur vitae metus magna.<br>Suspendisse molestie tellus vel ante scelerisque, nec lobortis turpis aliquam.</div>
+
+            
+            <img class="sign-president-img" src="./img/president.png">
+                <div class="sign-president">SIGNATURE OF PRESIDENT</div>
+    
+                <img class="sign-director-img" src="./img/director.png">
+                <div class="sign-director">SIGNATURE OF DIRECTOR</div>
+    
+                <div class="org-logo">
+                    <img src="./img/org-logo.png">
+                </div>
+    
+                <!--<div class="org-name">Maha Mission Education and Career Council</div>
+                <div class="org-address">
+                    <p>Office No. 404, Vastu Viva, Besids Sanket Inn, Bhumkar Chowk</p>
+                    <p>Wakad, Pune, Maharashtra - 411057</p>
+                </div> -->
```

Comment:
- remove this comment is required ** class="org-name"** 
-  ** class="org-address"**

Reply:
changes done please check it